onnxruntime/core/providers/cuda/cuda_execution_provider.cc

```diff
@@ -963,6 +963,13 @@
 class ONNX_OPERATOR_VERSIONED_KERNEL_CLASS_NAME(kCudaExecutionProvider, kOnnxDomain, 12, 12, Dropout);
 class ONNX_OPERATOR_KERNEL_CLASS_NAME(kCudaExecutionProvider, kOnnxDomain, 12, Einsum);
 
+class ONNX_OPERATOR_VERSIONED_TYPED_KERNEL_CLASS_NAME(kCudaExecutionProvider, kOnnxDomain, 12, 12, float, ArgMax);
```

Comment:
>class ONNX_OPERAT [](http://example.com/codeflow?start=0&length=17)

Typically, we also want to adjust ROCm registrations at the same time. Is this needed here?

Reply:
done